src/Http/Response/Format/Json.php

```diff
@@ -97,6 +97,11 @@ protected function morphToArray($value)
      */
     protected function encode($content)
     {
-        return json_encode($content);
+        $encodedString = json_encode($content);
+        if ($encodedString === false) {
+            throw new \ErrorException('Error encoding data in JSON format: '.json_last_error());
```

Comment:
You should import `\ErrorException` and add a `@throws \ErrorException` docblock to this method